.travis.yml

```diff
@@ -0,0 +1,52 @@
+language: python
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.cache/matplotlib
+
+python:
+  - 2.7
+  - 3.5
+  - 3.6
+    env: BUILD_DOCS=1
```

Comment:
I'd remove this (it may be the source of the travis issue) and just parse the `$TRAVIS_PYTHON_VERSION` env var.